src/sass/components/_advantages.scss

```diff
@@ -85,7 +86,98 @@
     }
   }
   img {
-    width: 360px;
+    min-width: 330px;
     height: 100%;
   }
 }
+
+@media screen and (min-width: 768px) {
+  .advantages {
+    &__container {
+      padding: 66px 0 140px 36px;
+      max-width: 700px;
+    }
+    &__list {
+      @include flex-display(row, 66px, center, start);
+    }
+    &__title {
+      line-height: 55px;
+      @include font-type(normal, header-height, little, medium);
+      margin-bottom: 16px;
+
+      &--vanilla-img::before {
+        margin-bottom: 36px;
+      }
+      &--apple-img::before {
+        margin-bottom: 36px;
+      }
+      &--heart-img::before {
+        margin-bottom: 36px;
+      }
+    }
+    &__text {
+      @include font-type(bold, text-small, medium, null);
+    }
+  }
+  .gallery {
+    &__container {
+      padding-left: 50px;
+      padding-right: 50px;
+      max-width: 700px;
+    }
+
+    &__img {
+      min-width: 786px;
+      min-height: 264px;
+    }
+    img {
+      min-width: 660px;
+      height: 100%;
+    }
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .advantages {
+    &__container {
+      padding: 39px 0 139px 115px;
+      max-width: 1050px;
+      margin: 0;
+    }
+    &__list {
+      @include flex-display(row, 50px, center, start);
+    }
+    &__title {
+      line-height: 55px;
+      @include font-type(normal, header-large, little, medium);
+      margin-bottom: 15px;
+
+      &--vanilla-img::before {
+        margin-bottom: 40px;
+      }
+      &--apple-img::before {
+        margin-bottom: 40px;
+      }
+      &--heart-img::before {
+        margin-bottom: 40px;
+      }
+    }
+    &__text {
+      @include font-type(bold, text-normal, medium, small);
```

Comment:
dla PC text powinien byc szerszy z kolei: 330px.
(mam na myśli wszystkie elementy listy, więc jeżeli img masz ogarniczone to tam trzeba rozszerzyć do 330px)